### PR TITLE
Enable docker support

### PR DESCRIPTION
**Pull request by johnsonw**
- Enable docker support

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

**Comment by johnsonw** [comment]
I'm not sure about this codefactor issue. It's recommending that I use `apt-get` instead of `apt`, but line 8 is using`apt-get`.

**Comment by jgrund** [comment]
I think it's counting the continuation on line 9.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM rust:1.39 as builder
+WORKDIR /build
+COPY . .
+RUN cargo build -p device-aggregator --release
+
+FROM rust:1.39
+COPY --from=builder /build/target/release/device-aggregator /usr/local/bin
+RUN apt-get update \
```
Comment (jgrund) [inline Dockerfile]: Is this needed, doesn't seem like apt is being used for anything.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM rust:1.39 as builder
+WORKDIR /build
+COPY . .
+RUN cargo build -p device-aggregator --release
+
+FROM rust:1.39
+COPY --from=builder /build/target/release/device-aggregator /usr/local/bin
+RUN rm -rf /var/lib/apt/lists/*
```
Comment (jgrund) [inline Dockerfile]: Nit: don't really need this either.

File: device-aggregator/src/main.rs [hunk 3]
```diff
@@ -30,11 +31,18 @@ fn main() -> Result<(), aggregator_error::Error> {
 
     let log = warp::log("device_aggregator");
 
+    let host = env::var("PROXY_HOST").expect("PROXY_HOST environment variable is required.");
```
Comment (jgrund) [inline device-aggregator/src/main.rs]: Is this being set correctly in both docker and non-docker contexts?

File: Dockerfile (hunk 2 above)
Comment (johnsonw) [reply Dockerfile]: Removed.

File: device-aggregator/src/main.rs (hunk 3 above)
Comment (johnsonw) [reply device-aggregator/src/main.rs]: Thinking about this more, it seems like it should be set to 127.0.0.1 in the event that it is not set.

**Pull request merged**